ovs/extensions/services/service.py

```diff
@@ -51,7 +57,21 @@ def __getattr__(cls, item):
                         else:
                             raise RuntimeError('The ServiceManager is unrecognizable')
                     elif 'systemd' in init_info:
-                        ServiceManager.ImplementationClass = Systemd
+                        if HAS_FLEET_CLIENT is True:
+                            try:
+                                if ServiceManager.has_fleet():
+                                    if ServiceManager._is_fleet_running_and_usable():
```

Comment:
if ServiceManager._is_fleet_running_and_usable() is False:
    ServiceManager.setup_fleet()